UserService/UserService.Application/UseCases/UserUseCases/GetUserInfoUseCase/GetUserInfoRequestHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿using AutoMapper;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using UserService.Application.DTO;
+using UserService.Domain.Entities;
+using UserService.Domain.Exceptions;
+
+namespace UserService.Application.UseCases.UserUseCases.GetUserInfoUseCase;
+
+internal class GetUserInfoRequestHandler(UserManager<AppUser> userManager, IMapper mapper)
+	: IRequestHandler<GetUserInfoRequest, UserInfoDTO>
+{
+	public async Task<UserInfoDTO> Handle(GetUserInfoRequest request, CancellationToken cancellationToken)
+	{
+		var user = await userManager.FindByIdAsync(request.userId);
+        if (user is null)
+        {
+            throw new NotFoundException("No such user");
+        }
```

Comment:
conditional statements should be followed and preceded by blank line. Fix tabulation